integ/test_init/test-guide.md

```diff
@@ -0,0 +1,107 @@
+###  Integration Test for Init Process of Fluent Bit on ECS
+
+Following this instruction, you will take the integration test for init process and get the test result
+
+
+
+#### Requirements
+
+1. [install AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
+2. [set up AWS account credential](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-quickstart.html)
+3. [install AWS CDK](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html)
+
+
+
+#### Deploy test resources
+
+* Create some Fluent Bit config files locally  (you can use any config files that are valid).
+
+* Upload config files to S3 and get ARNs.
+
+* Replace your ARNs in the `/integ_test/integ_test/integ_test_stack.py`.
+
+  **example:**
+
+  ```python
+  # change your ARNs here
+  arn1 = "arn:aws:s3:::yourBucket/dummy-input.conf"
+  arn2 = "arn:aws:s3:::yourBucket/dummy-filter.conf"
+  arn3 = "arn:aws:s3:::yourBucket/dummy-parser.conf"
+  arn4 = "arn:aws:s3:::yourBucket/dummy-s3-output.conf"
+          
+  
+  environment={ 
+              "aws_fluent_bit_init_s3_1": arn1,
+              "aws_fluent_bit_init_s3_2": arn2,
+              "aws_fluent_bit_init_s3_3": arn3,
+              "aws_fluent_bit_init_s3_4": arn4,
+              },
+  ```
+  
+* Deploy the CDK project.
+
+  ```shell
+  cd WORKSPACE/aws-for-fluent-bit/integ/test_init/integ_test
+  cdk synth
+  cdk deploy IntegTestStack
+  ```
+
+
+
+#### Check the test result
+
+* Go to ECS console to get cluster name and task-id we deployed.
+
+* Use [ECS Exec](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-exec.html) to directly interact with container.
+  ```shell
+  # replace the cluster-name and task-id to yours
+  
+  aws ecs execute-command --cluster Cluster-name \
+  --task task-id \
+  --container log_router \
+  --interactive \
+  --command "/bin/sh"
+  ```
+
+* Check the result.
+
+​		**example:**
+
+```shell
+# suppose we upload four config files to S3
+dummy-input.conf
+dummy-filter.conf
+dummy-parser.conf
+dummy-s3-output.conf
+
+# 1. go to the fluent-bit-init-s3-files folder to check the result
+cd /init/fluent-bit-init-s3-files
+ls
+# all config files you upload to S3 should be here, and the file content should be same
+
+
+# 2. check the main fluent bit config file
+cat /init/fluent-bit-init.conf
```

Comment:
So you don't need to do this since your internship is almost over, and I'd rather you focus on the docs first. But I think the ideal here would be if the user does not need to exec into the task. 

I have an idea of a better way. If you run another container in the task, it can mount these init directories from the fluent bit container into itself: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_data_volumes.html

And then, that container can just be a simple script that runs `ls` and `cat` on that directly and emits the file contents to stdout, and then those logs can be sent by Fluent Bit to CloudWatch. And then you could provide a script to grab the logs from CW so the user can check the contents. Our integ tests have example code for this. Or if that's too much work you can have them manually check the CW Logs.

Reply:
I would like to try this.
I think we can merge this version first? I've tested it on a colleague's computer and it works fine.